Storage/src/Connection/ServiceDefinition/storage-v1.json

```diff
@@ -1373,12 +1412,7 @@
         "description": "The entity holding the permission. Can be user-userId, user-emailAddress, group-groupId, group-emailAddress, allUsers, or allAuthenticatedUsers.",
         "required": true,
         "location": "path"
-       },
-       "provisionalUserProject": {
-        "type": "string",
-        "description": "The project to be billed for this request if the target bucket is requester-pays bucket.",
-        "location": "query"
-       },
```

Comment:
This makes me nervous.... could this potentially break backwards compatibility? Even more baffling, the api reference documentation [still includes this parameter](https://cloud.google.com/workflows/docs/reference/googleapis/storage/v1/bucketAccessControls/get). So we definitely should try to get more context before removing these.

Reply:
Now, that you've raised this, I observed `zoneAffinity` is also a parameter similarly removed from the service definition. I wonder why dont our tests capture this scenario.

Reply:
I've added a comment for `zoneAffinity`. This one we have discovered from the Storage team is a parameter for a feature that was never available in production, and so there are no compatibility concerns with removing it.